backend/src/database/entities/personnel.entity.ts

```diff
@@ -36,15 +38,15 @@ export class PersonnelEntity extends BaseEntity {
   @Column({type: 'date', name: 'date_joined'})
   dateJoined: Date;
 
-  @Column({
-    name: 'work_location',
-    type: 'varchar',
-    length: 100,
-  })
-  workLocation: string;
-
-  @Column({ name: 'region', type: 'enum', enum: Region, enumName: 'region' })
-  region: Region;
+  @JoinColumn([
+    {
+      name: 'workLocation',
```

Comment:
Can we keep db column names with underscores?
And since this is a reference, I think `work_location_id` would be more proper